.changes/next-release/feature-configure-74317.json

```diff
@@ -0,0 +1,5 @@
+{
+  "type": "feature",
+  "category": "configure",
+  "description": "Added the configure list-profiles command. See `#3735 <https://github.com/aws/aws-cli/issues/3735>`__."
```

Comment:
For some of these commands, it may be worth adding the literal notation so it gets rendered to code when the rst document is formed.